### PR TITLE
Fixed typo 0var to var

### PR DESCRIPTION
**Pull request by malavikakoppula**
## Description

Possible regression issue introduced in [THEMES-989: Add Themes JSON files to blocks by vgalatro · Pull Request #1631 · WPMedia/arc-themes-blocks](https://github.com/WPMedia/arc-themes-blocks/pull/1631/files) 

Seeing uses of 0var( where I assume it should be 0 var

## Jira Ticket

- [THEMES-1076](https://arcpublishing.atlassian.net/browse/THEMES-1076)

## Acceptance Criteria

`0var` should be `0 var`

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-1076]: https://arcpublishing.atlassian.net/browse/THEMES-1076?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/lead-art-block/themes/news.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
 						"display": "none"
```
Comment (vgalatro) [inline blocks/lead-art-block/themes/news.json:33]: Github isn't letting comment on the exact line, but on line 13 above there's two 0's without a space: `"padding": "00 var(--global-spacing-2) 0"`. It should be `"padding": "0 0 var(--global-spacing-2) 0"`. We should look for similar issues in the json files and fix them. The only way I can think this happened in the first place was the extension that converted the JS to JSON mishandled these strings.

**Review by vgalatro — CHANGES_REQUESTED** [review]
The changes made look good, but I found another issue that should be fixed. There are some places with `00` in the property string, and those will need to be corrected as well.

File: blocks/lead-art-block/themes/news.json (hunk 1 above)
Comment (malavikakoppula) [reply blocks/lead-art-block/themes/news.json:33]: Thanks @vgalatro, Fixed it in [ddaa22c](https://github.com/WPMedia/arc-themes-blocks/pull/1643/commits/ddaa22c42228792bef6e603144ecbe48b82123a2). Let me know if any more changes needed.

**Review by vgalatro — APPROVED** [review]
Thanks for fixing all of those, approved!

**Pull request merged**